orchagent/routeorch.cpp

```diff
@@ -1870,6 +1870,11 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
                         return false;
                     }
                 }
+                else if (m_neighOrch->isNextHopFlagSet(nexthop, NHFLAGS_IFDOWN))
```

Comment:
Just thinking if it impact tunnel route in dualtor case if NHFLAG is down? @Ndancejic , can you check this change?

Reply:
I think tunnel routes should be safe, I believe line 1839 would catch interface nexthops. also I think routes to tunnel nh get skipped in doTask:

```
/* skip direct routes to tun0 */
else if (alsv[0] == "tun0")
{
    it = consumer.m_toSync.erase(it);
}
```